doc/src/site/sphinx/deployment/sw_google_cloud_dataproc.rst

```diff
@@ -2,180 +2,135 @@ Running Sparkling Water on Google Cloud Dataproc
 ------------------------------------------------
 
 This section describes how to run Sparkling Water on `Google Cloud Dataproc <https://cloud.google.com/dataproc/docs/concepts/overview>`__. 
+It is meant to get you up and running with Sparkling Water on Google Cloud Dataproc as fast as possible so you can try it out.
+For further usage and productionizing some adjustments like `initialization actions <https://cloud.google.com/dataproc/docs/concepts/configuring-clusters/init-actions>`__ will be required.
+In this tutorial we will use Dataproc image version 2.0-debian10 which has Spark 3.1 and Scala 2.12.
 
-Perform the following steps to start Sparkling Water ``H2OContext`` on Cloud Dataproc. 
+1. Install the `Google Cloud SDK <https://cloud.google.com/sdk/docs/install>`__. Login to your account.
+2. Download `Sparkling water <http://h2o-release.s3.amazonaws.com/sparkling-water/spark-3.1/3.34.0.3-1-3.1/index.html>`__.
+3. Create a Google Cloud Dataproc cluster.
 
-**Note**: Be sure to allocate enough resources to handle your data. As a minimum recommendation, you should allocate 4 times memory of the size of your data in H2O.
+.. code:: bash
 
-1. Install the `Google SDK <https://cloud.google.com/sdk/gcloud/>`__.
+    DATAPROC_CLUSTER_NAME='sparkling-water-test'
+    GCP_REGION='europe-central2'
 
-2. Install `Daisy <https://googlecloudplatform.github.io/compute-image-tools/daisy-installation-usage.html>`__. The path for Daisy will be required in a later step.
+    gcloud dataproc clusters create $DATAPROC_CLUSTER_NAME \
+    --region $GCP_REGION \
+    --image-version 2.0-debian10 \
+    --num-workers 3 \
+    --properties='^#^dataproc:pip.packages=tabulate==0.8.3,requests==2.21.0,future==0.17.1'
 
-3. After Daisy is installed, download and save Google's `generate_custom_image.py <https://github.com/GoogleCloudPlatform/cloud-dataproc/blob/master/custom-images/generate_custom_image.py>`__. This custom image will be referenced in a later step.
+.. content-tabs::
 
-4. Copy and save the following customization script. This will be required in a later step.
+    .. tab-container:: Scala
+        :title: Scala
 
- ::
+        Set the variables.
 
-	#!/bin/bash
+        .. code:: bash
 
-	apt-get update
-	apt-get install -y python-dev python-pip jq
+            # we need spark jars to compile the example
+            SPARK_JARS=$(echo "$SPARK_HOME"/jars/*.jar | tr ' ' ':')
+            SPARKLING_WATER_JAR='sparkling-water-assembly_2.12-3.36.0.1-1-3.1-all.jar'
```

Comment:
This version hasn't been release yet. Can we use place holders for version parts? see https://github.com/h2oai/sparkling-water/blob/master/doc/build.gradle#L43-L57